src/pages/profile.tsx

```diff
@@ -0,0 +1,64 @@
+import { useSession } from 'next-auth/react';
+import { useEffect, useState } from 'react';
+import { SignOutButton } from '../components/SignOutButton';
+import { Image } from '@chakra-ui/react';
+
+export default function AnotherPage() {
+  const { data: session, status } = useSession();
+  const [numAbsences, setNumAbsences] = useState(null);
```

Comment:
Type safety for num absences
const [numAbsences, setNumAbsences] = useState<number | null>(null);